src/test/java/com/datapipe/jenkins/vault/configuration/VaultConfigurationSpec.java

```diff
@@ -84,13 +84,13 @@ public void shouldHandleNull() {
 
     @Test
     public void shouldNotStoreTrailingSlashesInUrl() {
-        VaultConfiguration parent = new VaultConfiguration("http://vault-url.com/", null, false);
+        VaultConfiguration parent = new VaultConfiguration("http://vault-url.com/", null, false, "mynamespace");
```

Comment:
please revert all of the test changes where you added `"mynamespace"` and create an actual test using the setter.